src/app/shared/ui/menu/menu.component.html

```diff
@@ -0,0 +1,54 @@
+<nav class="wrapper">
+  <div class="top-section">
+    <div class="toggle-mode-container">
+      <mat-slide-toggle></mat-slide-toggle>
+      <p>Dark mode</p>
+    </div>
+    <button (click)="closeMenu()" class="close-button">
+      <mat-icon fontIcon="close" [style]="{ padding: '10px' }"></mat-icon>
+    </button>
+  </div>
+
+  <div *ngIf="user.userData as userData" class="user-info-container">
```

Comment:
moze dodajmy tutaj user?.userData , nizej w role podobnie

Reply:
nie bardzo rozumiem. możesz rozwinąć myśl? 🤔

Reply:
sprawdzilam, ze jeśli user byłby nullem, to wywali nam ponizszy błąd;  a dodając '?' możemy się przed tym zabezpieczyć

![image](https://user-images.githubusercontent.com/87782933/217016916-13f44724-a02f-4306-9a5b-8cb349d5b51e.png)

Reply:
oki, poprawione